### PR TITLE
[RELEASE-1.17] Add old eventing CRDs patch

### PR DESCRIPTION
**Pull request by devguyio**
- Add old versions to eventing CRDs with both served and storage set to False

### CODE REVIEW

File: openshift-knative-operator/cmd/operator/kodata/knative-eventing/0.23.0/1-eventing-crds.yaml [hunk 1]
```diff
@@ -201,6 +245,176 @@ spec:
         - name: Reason
           type: string
           jsonPath: ".status.conditions[?(@.type==\"Ready\")].reason"
+    - !!merge <<: *version
+        name: v1alpha2
+        served: false
+        storage: false
+        schema:
+          openAPIV3Schema: &openAPIV3Schema
+            type: object
```
Comment (matzew) [inline openshift-knative-operator/cmd/operator/kodata/knative-eventing/0.23.0/1-eventing-crds.yaml:254]: I get:

```
error: error parsing 1-eventing-crds.yaml: error converting YAML to JSON: yaml: line 248: did not find expected key
```

Comment (devguyio) [reply openshift-knative-operator/cmd/operator/kodata/knative-eventing/0.23.0/1-eventing-crds.yaml:254]: I'll try again, but those files were generated from the patch

Comment (devguyio) [reply openshift-knative-operator/cmd/operator/kodata/knative-eventing/0.23.0/1-eventing-crds.yaml:254]: It's working on my machine. Which yq version do you have?

**Comment by devguyio** [comment]
```
level=fatal msg="failed to fetch Cluster: failed to generate asset \"Cluster\": failed to create cluster: failed to apply Terraform: failed to complete the change"
```
Infra
/retest

**Comment by devguyio** [comment]
wth! It says needs lgtm while it has it!
/lgtm

**Review by matzew — APPROVED** [review]
/lgtm
/approve
/go-in
/joke

**Pull request merged**